src/pages/home/report/ReportActionsView.tsx

```diff
@@ -295,85 +305,118 @@ function ReportActionsView({
      * Retrieves the next set of report actions for the chat once we are nearing the end of what we are currently
      * displaying.
      */
-    const loadOlderChats = useCallback(() => {
-        Log.info(
-            `[ReportActionsView] loadOlderChats ${JSON.stringify({
-                isOffline: network.isOffline,
-                isLoadingOlderReportActions,
-                isLoadingInitialReportActions,
-                oldestReportActionID: oldestReportAction?.reportActionID,
-                hasCreatedAction,
-                isTransactionThread: !isEmptyObject(transactionThreadReport),
-            })}`,
-        );
+    const loadOlderChats = useCallback(
```

Comment:
I feel like these names here are misleading since we are not loading older or newer chats/ reports but only messages. NAB